code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -169,6 +170,19 @@
 		current_potion = O
 		to_chat(user, "<span class='notice'>You load [O] in the console's potion slot[replaced ? ", replacing the one that was there before" : ""].</span>")
 		return
+	else if(istype(O, /obj/item/slime_extract))
+		var/obj/item/slime_extract/E = O
+		if(!already_researched[E.type])
+			playsound(src, 'sound/machines/ping.ogg', 50, 3, -1)
+			visible_message("<span class='notice'>You insert the [E] into a slot on the [src]. It pings and prints out some research notes worth [E.research] points!</span>")
```

Comment:
Remove uses of "the" infront of vars. We handle those automatically when the name is used. Else it comes out as The the slime extract into the the console